### PR TITLE
New package: exaworks

### PR DESCRIPTION
**Pull request by andre-merzky**
This PR provides a meta package to bundle the exaworks related packages into one.  I would appreciate feedback on the approach: is that the right way to define meta-packages?  Thanks!

### CODE REVIEW

File: var/spack/repos/builtin/packages/exaworks/package.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Exaworks(Package):
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/exaworks/package.py]: Hi @andre-merzky, one change that would be worthwhile is to inherit from `BundlePackage` (like `xsdk`). That handles:

* `has_code = False` (so you can remove that here)
* Also indicates that there are no phases to run when building the package (i.e. sets `phases = []`)

Comment (andre-merzky) [reply var/spack/repos/builtin/packages/exaworks/package.py]: Thanks for the feedback, @scheibelp !  The package now inherits from `BundlePackage` as advised :-)

File: var/spack/repos/builtin/packages/exaworks/package.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    depends_on('flux-sched',              type=('build', 'run'))
+
+    depends_on('py-parsl',                type=('build', 'run'))
+
+    depends_on('py-radical-gtod',         type=('build', 'run'))
+    depends_on('py-radical-utils',        type=('build', 'run'))
+    depends_on('py-radical-saga',         type=('build', 'run'))
+    depends_on('py-radical-pilot',        type=('build', 'run'))
+    depends_on('py-radical-entk',         type=('build', 'run'))
+
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/exaworks/package.py]: flake is complaining about this blank line.

```
  var/spack/repos/builtin/packages/exaworks/package.py:33: [W391] blank line at end of file
  flake8 found errors
```

Once that is resolved, this looks good!

Comment (andre-merzky) [reply var/spack/repos/builtin/packages/exaworks/package.py]: Another repo, another set of rules :-)  Thanks for noting this, this is fixed now!

**Review by scheibelp — APPROVED** [review]

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**